### ISSUE TITLE
Default rollup config

### ISSUE THREAD

**Issue by CallMeLaNN** [rafgraph/rollpkg#2]
Im trying this. So far ok. I got some warning that actually can be fixed if I can overwrite the rollup config.

```
'fs/promises' is imported by src/index.ts, but could not be resolved – treating it as an external dependency
preferring built-in module 'url' over local alternative at '/Users/foo/Documents/my-project/node_modules/url/url.js', pass 'preferBuiltins: false' to disable this behavior or 'preferBuiltins: true' to disable this warning
```

I should be able to add `fs/promises` into rollup external or maybe we can use `@rollup/plugin-node-resolve`?

Regarding 'url', I'm using builtin fileURLToPath but somehow conflicted with url from npm that is required by rollpkg dependencies. We can make preferBuiltins: true as default.

Anyway it is awesome if rollup config can be extended in future.

**Comment by rafgraph** [comment]
Hi, thanks for the report. I added `preferBuiltins: true` to the `@rollup/plugin-node-resolve` config and just published v0.4.3, let me know if this helps.

I'm not sure about `'fs/promises'`, is your repo on GitHub? I can take a look.

Regarding extending the rollup config, I like the idea, but not sure what the best approach is. Rollpkg uses Rollup's js api, see [here](https://github.com/rafgraph/rollpkg/blob/main/src/rollupBuilds.ts#L60-L108). I'm open to ideas for how this could work. Thanks.

**Comment by CallMeLaNN** [comment]
I just run this in index.ts locally

```
import { stat } from "fs/promises"
stat("somefile.json").then((s) => console.log(s))
```

since it is available in NodeJS builtins, short version of doing promisify I think, this is no big deal.

Let's see if we can do something on Rollpkg rollup config, so far important plugins for me is there already. Anyway thanks.
